module/proto/gravity/v1/gravity.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+package gravity.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/gravity-bridge/module/x/gravity/types";
+
+// EthereumEventVoteRecord is an event that is pending of confirmation by 2/3 of
+// the signer set. The event is then attested and executed in the state machine
+// once the required threshold is met.
+message EthereumEventVoteRecord {
+  google.protobuf.Any event = 1
+      [ (cosmos_proto.accepts_interface) = "EthereumEvent" ];
+  repeated string votes = 2;
+  bool accepted = 3;
+}
+
+// LatestEthereumBlockHeight defines the latest observed ethereum block height
+// and the corresponding timestamp value in nanoseconds.
+message LatestEthereumBlockHeight {
```

Comment:
Come to think of it, this is confusingly named. It is not the latest ethereum block height, but simply the last height that Gravity module has knowledge of. The old name `LastObservedEthereumBlockHeight` is more accurate. We have removed "Observed" from the naming scheme for the most part, but maybe it is fitting here.